code_samples/sql-basic-usage.js

```diff
@@ -19,8 +19,12 @@ const { Client, SqlColumnType } = require('hazelcast-client');
 
 (async () => {
     try {
-        const client = await Client.newHazelcastClient();
-        const mapName = 'myMap';
+        const client = await Client.newHazelcastClient({
+            properties: {
+                'hazelcast.logging.level': 'OFF'
+            }
+        });
+        const mapName = 'myMap' + Math.floor(Math.random() * 10000);
```

Comment:
I think it would be better to use a constant map name here, since the user may want to inspect this map after running the example code. Other samples seems to have maps with random names as well.

Reply:
You can use `CREATE OR REPLACE MAPPING` to replace the mapping if it already exists so the user doesn't have to delete the mapping/restart the cluster to run the same example again.

Reply:
> I think it would be better to use a constant map name here, since the user may want to inspect this map after running the example code. Other samples seems to have maps with random names as well.

It is random since we run all code samples with a cluster in code sample checker. So if the map names are the same they will break with each other. I do not like to do that because of this. We can change code sample checker code to use a new cluster for each code sample. I think we should add this task to jira backlog. 

> You can use CREATE OR REPLACE MAPPING to replace the mapping if it already exists so the user doesn't have to delete the mapping/restart the cluster to run the same example again.


Since the map name is random it won't be needed but it won't harm so doing it.

Reply:
Alternatively we can log map names to console after creating its name.